client/landing/gutenboarding/onboarding-block/vertical-select/index.tsx

```diff
@@ -0,0 +1,129 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { useState, useCallback, useRef } from 'react';
+import { __ as NO__ } from '@wordpress/i18n';
+import { useDispatch, useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import Suggestions from '../../../../components/suggestions';
+import { STORE_KEY } from '../../store';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+import { SiteVertical, isFilledFormValue } from '../../store/types';
```

Comment:
I'm not a big fan of these import headers in general, but types is under the wrong one 🙂